src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -724,3 +724,9 @@ def test_wait_for_event_repeated_capabilities(session, channels):
     channels_session = session.channels[channels]
     with channels_session.initiate():
         channels_session.wait_for_event(nidcpower.Event.SOURCE_COMPLETE)
+
+
+def test_no_underlying_rlock(session):
```

Comment:
This is a good test to run as part of "dev testing" but don't think it makes a good system test to be used for regression. Outside of this PR, the test doesn't provide sufficient context or utility for maintainers. I'd say remove it.